### PR TITLE
Validations made

### PR DESCRIPTION
**Pull request by tllach**

### CODE REVIEW

File: Homepage.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 today = date.today()
 default_date_yesterday = today - timedelta(days=1)
```
Comment (Katkat04) [inline Homepage.py]: Are those two lines still necessaries?

File: Homepage.py [hunk 2]
```diff
@@ -61,22 +66,32 @@ def run_query(query):
...
-if buttonSearch:
-    #Funcion que retorna los paises seleccionados en forma de query
-    sentence = createSentence(country)
-    
-    rows = run_query(f"select * from DatosCovid as dc where dc.Date >= '{start_date}' and dc.Date <= '{end_date}' and dc.Location IN {sentence};")
-    # Print results.
-    for row in rows:
-        st.write(f"{row[0]} - {row[1]} - {row[2]} - {row[3]} - {row[4]}")
+def isAllCorrect() -> bool:
+    return len(country) > 0
```
Comment (Katkat04) [inline Homepage.py:76]: is this the only validation the app need?

**Review by Katkat04 — CHANGES_REQUESTED** [review]

File: Homepage.py (hunk 1 above)
Comment (tllach) [reply Homepage.py]: Actually, I didn't catch that but I'll delete it.

File: Homepage.py (hunk 2 above)
Comment (tllach) [reply Homepage.py:76]: Yes, because the other fields have an initial value so if the user doesn't change any of them it won't get a error

**Review by Katkat04 — APPROVED** [review]
All good

**Pull request merged**